packages/fiori/src/themes/sap_belize_hcw/Login-parameters.css

```diff
@@ -0,0 +1,5 @@
+:root {
+    --ui5-login-header-background-color: var(--sapButton_Emphasized_BorderColor);

```

Comment:
These are the same as ones in the base. When there is no difference, we don't need to introduce theme specific parameters file, but just reference the base as you did for other themes.

```css
/* packages/fiori/src/themes/base/Login-parameters.css */
--ui5-login-header-background-color: var(--sapButton_Emphasized_BorderColor);
--ui5-login-error-message-text-color: var(--sapErrorBorderColor);
--ui5-login-error-message-background-color: var(--sapErrorBackground);
   ```